docs/source/upgrade_to_newest_version.md

```diff
@@ -0,0 +1,186 @@
+# Considerations for getting to v2.0
+
+## Chaincode lifecycle
+
+For more information about the new chaincode lifecycle, check out [Chaincode for operators](./chaincode4noah.html).
+
+It is a best practice to upgrade all of the peers on a channel before enabling
+the `Channel` and `Channel/Application` capabilities that enable the new chaincode
+lifecycle to function. Note that any peers that are not at v2.0 will crash after
+enabling either capability, while any ordering nodes that are not at v2.0 will
+crash after the `Channel` capability has been enabled. This crashing behavior is
+intentional as it prevents the node from rejecting the config block (given that
+it contains a value it does not understanding) when nodes at the appropriate
+node level would accept it.
+
+After the `Application` capability has been updated to `V2_0`, you must use
+the v2.0 lifecycle procedures to package, install, approve, and commit new
+chaincodes on the channel. As a result, make sure to be prepared for the new
+lifecycle before updating the capability.
+
+The new lifecycle defaults to using the endorsement policy configured in the
+channel config (e.g., a `MAJORITY` of orgs). Therefore this endorsement policy
+should be added to the channel configuration when enabling capabilities on the
+channel.
+
+For information about how to edit the relevant channel configurations to enable
+the new lifecycle, check out [Enabling the new chaincode lifecycle](./enable_cc_lifecycle.html).
+
+It is also a best practice to add an endorsement policy for each organization,
+which will also define how changes will be made at the organization level.
+
+## Chaincode shim changes
+
+Note that these changes only apply to chaincode written in Go.
+
+If you vendor the shim in v1.4 chaincode before making upgrades to the peers and
+channels, no action is needed for the chaincode. This is the recommended approach.
```

Comment:
I had to read this a few times to get it. Would something like this be clearer?

Before making upgrades to the peers and channels it is recommended that you vendor the shim in v1.4 chaincode, then no further action is needed for the chaincode.

Reply:
Yeah this sentence could definitely be better. I massaged it some.